packages/dscc-gen/src/files.ts

```diff
@@ -27,19 +25,35 @@ const ENCODING = 'utf8';
 const CURR_DIR = process.cwd();
 
 const fixFile = (templates: Template[]) => async (file: string) => {
-  const contents = await util.readFile(file, ENCODING);
+  const contents = util.readFile(file, ENCODING);
   const newContents = templates.reduce(
     (acc, {match, replace}) => acc.replace(match, replace),
     contents
   );
   return util.writeFile(file, newContents, ENCODING);
 };
 
+export const recursiveReaddirSync = async (
```

Comment:
It looks like this function shouldn't be async since everything in it is synchronous.

Reply:
yep - good catch.